<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrownView</class>
 <widget class="QWidget" name="CrownView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrownView</string>
  </property>
  <widget class="VitaWidget" name="vitaWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>295</width>
     <height>55</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>41</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Безпрагово изпиляване</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Прагово изпиляване</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Завинтване в/у имплант</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Циментиране в/у имплант</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="materialLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Материал:</string>
   </property>
  </widget>
  <widget class="RangeWidget" name="rangeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="materialEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">View/uiComponents/qt_derived/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>RangeWidget</class>
   <extends>QWidget</extends>
   <header>View/uiComponents/qt_derived/rangewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VitaWidget</class>
   <extends>QWidget</extends>
   <header>View/uiComponents/qt_derived/vitawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
